lighthouse/blueprints/plate_events.py

```diff
@@ -47,10 +48,16 @@ def create_plate_event() -> Tuple[Dict[str, Any], int]:
             broker.publish(message, routing_key)
             broker.close_connection()
             logger.info(f"Successfully published a '{event_type}' plate event message")
-            return ({"errors": []}, HTTPStatus.OK)
+            success, messages = fire_callbacks(message)
+            if success:
+                return ({"errors": []}, HTTPStatus.OK)
+            else:
+                logger.error("Failed to update labwhere", messages)
+                return ({"errors": messages}, HTTPStatus.INTERNAL_SERVER_ERROR)
```

Comment:
Wondering if we should update LabWhere whether or not the message to the Event WH failed?

Reply:
Might be a good call, as those plates *will* have been picked at that point.

Reply:
Yeah, leaves us hopefully only one data fix to make rather than two, if something goes wrong